integration/containerd/confidential/agent_image.bats

```diff
@@ -106,6 +94,10 @@ setup() {
 	clear_kernel_params
 }
 
+get_kata_logs() {
```

Comment:
Here goes a prototype (feel free to use or not):
```
+# Check the logged messages on host have a given message.
+# Parameters:
+#      $1 - the message
+#
+#
+# Note: get the logs since the global $test_start_time.
+#
+assert_logs_have() {
+       local str="$1"
+       journalctl -x -t kata --since "$test_start_time" | grep "$str"
+}
+
 setup() {
+       test_start_time="$(date +'%F %T')"
        sandbox_name="kata-cc-busybox-sandbox"
        pod_config="${FIXTURES_DIR}/pod-config.yaml"
        pod_id=""
@@ -147,7 +160,8 @@ setup() {
        create_test_pod
 
        assert_container_fail "$container_config"
-       grep 'Signature for identity .* is not accepted' "$console_file"
+       assert_logs_have "Signature for identity"
+       #grep 'Signature for identity .* is not accepted' "$console_file"
 }
```
ah, I think it will need to handle regex in the message string...

Reply:
Great - thanks Wainer. I originally had the `--since` in but think I messed up something else, so removed it. I'll look at implementing it as you suggest and see how I get on

Reply:
I think it's working with the `--since` logic now and the tests are all passing

Reply:
Thanks for working on that too!